pom.xml

```diff
@@ -76,7 +96,35 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>credentials-binding</artifactId>
-      <version>1.7</version>
+      <version>1.16</version>
+    </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>jackson2-api</artifactId>
+      <version>2.8.11.3</version>
+    </dependency>
+    <dependency>
+      <groupId>org.jenkinsci.plugins</groupId>
+      <artifactId>pipeline-model-extensions</artifactId>
```

Comment:
Much as I loathe `optional` dependencies, maybe it makes sense here—this dependency pulls in a fat subtree including `workflow-cps` and `workflow-job`. Of course better might be for the extension API to not depend on these things to begin with. CC @abayer